Exiled.API/Features/Spawn/SpawnPosition.cs

```diff
@@ -0,0 +1,46 @@
+// -----------------------------------------------------------------------
+// <copyright file="SpawnPosition.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.API.Features.Spawn
+{
+    using PlayerRoles;
+    using UnityEngine;
+
+    /// <summary>
+    /// Represents a spawn location for a <see cref="Roles.Role"/>.
+    /// </summary>
+    public class SpawnPosition
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="SpawnPosition"/> class.
+        /// </summary>
+        /// <param name="roleType">The <see cref="RoleTypeId"/> this spawn is for.</param>
+        /// <param name="position">The <see cref="Vector3"/> position of the spawn.</param>
+        /// <param name="horizontalRotation">The horizontal rotation of the spawn.</param>
+        internal SpawnPosition(RoleTypeId roleType, Vector3 position, float horizontalRotation)
```

Comment:
```suggestion
        public SpawnPosition(RoleTypeId roleType, Vector3 position, float horizontalRotation)
```
It could be useful to make it public, someone could use this class in their plugins.